examples/customresources/module.json

```diff
@@ -0,0 +1,72 @@
+{
+	"modules": [
+		{
+			"name": "AcmeModule",
+			"path": "examples/customresources/module/module"
```

Comment:
Can we make this a tad more descriptive?
path_binary? 
or something?

Reply:
Changing it to "executable_path" to follow the convention we have elsewhere (e.g. label_path, model_path, serial_path, etc.)